compiler/ballerina-backend-jvm/src/main/ballerina/compiler_backend_jvm/jvm_type_gen.bal

```diff
@@ -583,7 +618,7 @@ function addObjectAtatchedFunctions(jvm:MethodVisitor mv, bir:BAttachedFunction?
 # + mv - method visitor
 # + attachedFunc - object attached function
 function createObjectAttachedFunction(jvm:MethodVisitor mv, bir:BAttachedFunction attachedFunc,
-                                        bir:BObjectType objType) {
+                                      bir:BObjectType objType, string pkgName) {
```

Comment:
pkgName is not used?

Reply:
Removed this and few other unnecessary pkgName params in type creation functions with cb6db58.